Jasmine/Common/ViewModels/TimedViewModelProtocol.swift

```diff
@@ -0,0 +1,21 @@
+import Foundation
+
+/// A ViewModel that implements a timer, i.e. it is timed.
+protocol TimedViewModelProtocol: BaseViewModelProtocol {
```

Comment:
The current implementation works... but idk, just feels quite weird. Like VM contains the Timer object, yet the Timer object also contains the VM object containing it, as well as the VCDelegate owned by VM. That sounds kinda off isn't it? (like, check the TImer class)

Or is it normal?